### ISSUE TITLE
6.3 FPPOS upgrade requires power cycle to complete. 

### ISSUE THREAD

**Issue by ghormann** [FalconChristmas/fpp#1449]
**Describe the bug and Steps to reproduce**: 
A clear and concise description of what the bug is and the steps to be taken to reproduce. 
1. Start with 5.x, 6.x (including 6.3 or current) on Pi 3B or BBB. 
2. Use FPPOS to download and install 6.3 FPPOS (even on top of 6.3)
3. OS installs fine, and reboot starts but never finishes. 
4. Disconnecting power and power cycling device allows OS to boot.   

**Expected behavior**: 
Should reboot cleanly after upgrade.

**Additional context**: 
1. If monitor is attached (tested with PI) the color screen appears, but never see kernel boot message nor "Your IP IS" nor Login prompt.
2. These will display after power cycle.

**Comment by bloojoop** [comment]
Every upgrade we have done to 6.2 has behaved the same way.  a complete power cycle (sometimes 2 power cycles) are needed to regain access to the instance.  Post power cycle, it appears to be upgraded and functional but when controller are up on the roof, sometimes a power cycle is not easily accomplished.

**Comment by bloojoop** [comment]
I just did a fppos update on my k8-B to 6.3.  I am working remotely from the controller.  I watched it reboot by way of performing a continuous ping.  upon first reboot, it registered with dhcp as FPP even though the host name is configured as k8b.  I can ping it now but ssh does not work, it shows connection refused.  I also cannot get to browser as it also shows connection refused. In this state, I expect it will require a power cycle to come back to life.  
![image](https://user-images.githubusercontent.com/88390168/210276200-d7c2fd55-32f4-487e-a60d-2fed09b12801.png)
![image](https://user-images.githubusercontent.com/88390168/210276227-a4317ce6-6860-4f84-9ca5-852319af71ca.png)

**Comment by bloojoop** [comment]
Another quick note, after a power cycle, I was able to access my k8 again by both browser and ssh...After this power cycle, it still registered in dhcp as FPP.  The hostname on the controller is definitely still set to k8b
![image](https://user-images.githubusercontent.com/88390168/210279264-5fcb9299-a435-4be4-9c59-0472076479ea.png)
I performed another reboot, and now the hostname is registered in dhcp as expected.   
So in summary, to use hostname via dhcp/dynamic dns, After fppos upgrade, I had to power cycle once, then reboot a second time to get the hostname to register properly via dhcp.

**Comment by cybercop23** [comment]
Arggg... same here.. thanks for finding this. 5 controllers upgraded and they all had the same issue, had to power cycle to be able to get to them. I use hostnames to connect to all of them.

**Comment by darylc** [comment]
so I connected a monitor to my pi and did 6.3->6.3 upgrades.  It does reboots, but when it boots, it flicks back and forth between a blank screen with a flashing _ on the top left, and a totally bank screen every few seconds.

I also have a kulp oled board on this pi, which has a IP from my DHCP range on it rather than the static configured, and says FPPD is not running.  No ports are open on FPP at this point and none of the tty's show anything on the monitor...

**Comment by darylc** [comment]
Have run out of time tonight, documenting where I got to for tomorrow. Traced it down to /opt/fpp/scripts/init_pre_network

that seems to log to /home/fpp/media/logs/fpp_init.log 

For the hanging boot, the log says:
==========================================================================
Tue 13 Sep 2022 02:58:47 AM BST
FPP - Setting up for the Falcon Player on the Raspberry Pi platform
FPP - Checking for required directories
Performing boot action: settings
Synchronizing state of ntp.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable ntp
Job for ntp.service canceled.
ESC[0;1;31mFailed to set time zone: Refusing activation, D-Bus is shutting down.ESC[0m
ls: cannot access '/home/fpp/media/config/interface.w*': No such file or directory
ls: cannot access '/home/fpp/media/config/interface.e*': No such file or directory
net.ipv4.ip_forward = 0
Failed to reload-or-restart systemd-networkd.service: Transport endpoint is not connected
See system logs and 'systemctl status systemd-networkd.service' for details.
Failed to connect to bus: Connection refused
Synchronizing state of hostapd.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable hostapd
Failed to connect to bus: Connection refused
Failed to connect to bus: Connection refused
Failed to connect to bus: Connection refused
FPP - Clearing reboot flags
FPP - Sanity checking storage device setting

If I power cycle then it says:
==========================================================================
Tue 13 Sep 2022 02:58:50 AM BST
FPP - Setting up for the Falcon Player on the Raspberry Pi platform
FPP - Checking for required directories
ls: cannot access '/home/fpp/media/config/interface.w*': No such file or directory
ls: cannot access '/home/fpp/media/config/interface.e*': No such file or directory
net.ipv4.ip_forward = 0
Synchronizing state of hostapd.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable hostapd
FPP - Clearing reboot flags
FPP - Sanity checking storage device setting
FPP - Setting max IGMP memberships
FPP - Screen blanking deactivated

**Comment by darylc** [comment]
init_pre_network calls handle_boot_actions and it's the line 13: ApplySetting("ntp", "1"); that hangs the boot process.

Tracing that through /opt/fpp/www/common/settings.php, it runs:
      exec("sudo systemctl enable ntp");
        exec("sudo systemctl start ntp");

I don't think this is needed at all.  Commenting out the ApplySetting("ntp", "1") results in fpp starting up after the upgrade, ntp is running and listening on the correct IP addresses (eg the static IP from fpp config, not a dhcp address.

Going to sleep on this overnight before I commit a change to remove it.

**Comment by darylc** [comment]
This fix wont take effect until a new fppos file is released.
